okhttp-testing-support/src/commonMain/kotlin/okhttp3/TestUtilCommon.kt

```diff
@@ -27,3 +28,11 @@ expect val SYSTEM_FILE_SYSTEM: FileSystem
 expect fun getEnv(name: String): String?
 
 expect val isJvm: Boolean
+
+fun String(vararg codePoints: Int): String {
```

Comment:
I can't believe it's come to this...

Reply:
If you wish to make an apple pie from scratch, you must first invent the universe.

Reply:
🍎 + 🥧 = �